starter-code/css/main.css

```diff
@@ -6,3 +6,112 @@ Green: #00B172
 White: #FFF
 
 */
+
+body {
+  font-family: Helvetica,Arial,sans-serif;
+  font-weight: lighter;
+  margin: 0;
+}
+
+/*HEADER SECTION*/
+
+header {
+  padding: 10px 40px 0px 40px;
+}
+
+header img {
```

Comment:
Do you really want the logo to shrink or expand when the user resizes the window?